Libplanet.Tests/Net/Protocols/ProtocolTest.cs

```diff
@@ -82,39 +86,9 @@ public async Task Ping()
                 await transportA.MessageReceived.WaitAsync();
                 await Task.Delay(100);
 
-                Assert.Single(transportA.ReceivedMessages);
-                Assert.Single(transportB.ReceivedMessages);
-                Assert.Contains(transportA.AsPeer, transportB.Peers);
-            }
-            finally
-            {
-                await transportA.StopAsync(TimeSpan.Zero);
-                await transportB.StopAsync(TimeSpan.Zero);
-            }
-        }
-
-        [Fact(Timeout = Timeout)]
-        public async Task PingTwice()
```

Comment:
The test was to check each peer can receive and respond to `ping`/`pong`. Logic became to handshake before connection, it will resend another `ping` message when a `ping` message is received. Thus the `PingTwice` test became meaningless.